io.github.mpc_qt.mpc-qt.yml

```diff
@@ -0,0 +1,294 @@
+                dest: 3rdparty/glad
+                x-checker-data:
+                  type: anitya
+                  project-id: 300234
+                  url-template: https://github.com/Dav1dde/glad/archive/refs/tags/v$version.tar.gz
+
+          - name: libXpresent
+            buildsystem: autotools
+            sources:
+              - type: archive
+                url: https://xorg.freedesktop.org/archive/individual/lib/libXpresent-1.0.1.tar.xz
+                sha256: b964df9e5a066daa5e08d2dc82692c57ca27d00b8cc257e8e960c9f1cf26231b
+                x-checker-data:
+                  type: anitya
+                  project-id: 17166
+                  stable-only: true
+                  url-template: https://xorg.freedesktop.org/archive/individual/lib/libXpresent-$version.tar.xz
+
+          - name: nv-codec-headers
+            cleanup:
+              - '*'
+            no-autogen: true
+            make-install-args:
+              - PREFIX=/app
+            sources:
+              - type: archive
+                url: https://github.com/FFmpeg/nv-codec-headers/archive/refs/tags/n13.0.19.0.tar.gz
+                sha256: 86d15d1a7c0ac73a0eafdfc57bebfeba7da8264595bf531cf4d8db1c22940116
+                x-checker-data:
+                  type: anitya
+                  project-id: 223796
+                  stable-only: true
+                  url-template: https://github.com/FFmpeg/nv-codec-headers/archive/refs/tags/n$version.tar.gz
+
+          - name: ffmpeg
+            cleanup:
+              - /share/ffmpeg/examples
+            config-opts:
+              - --disable-debug
+              - --disable-doc
+              - --disable-programs
+              - --disable-static
+              - --disable-encoders
+              - --disable-muxers
+              - --enable-shared
+              - --enable-gnutls
+              - --enable-gpl
+              - --enable-version3
+              - --enable-encoder=png,libwebp,libjxl
+              - --enable-libv4l2
+              - --enable-libdav1d
+              - --enable-libfontconfig
+              - --enable-libfreetype
+              - --enable-libopus
+              - --enable-librsvg
+              - --enable-libvpx
+              - --enable-libmp3lame
+              - --enable-libwebp
+              - --enable-libjxl
+              - --enable-libxml2
+              - --enable-vulkan
+#             - --enable-libx264 (needed for encoding)
+            sources:
+              - type: archive
+                url: https://ffmpeg.org/releases/ffmpeg-7.1.tar.gz
+                sha256: 42a7dc0d1583885d1b8f6559fa7ce28f97acafea6803de6a8f73e3ba229348bd
+                x-checker-data:
+                  type: anitya
+                  project-id: 5405
+                  stable-only: true
+                  url-template: https://ffmpeg.org/releases/ffmpeg-$version.tar.gz
+            modules:
+              - name: libjxl
```

Comment:
this is in the runtime.

Reply:
ffmpeg didn't build without it.

Reply:
it is in the 6.8 runtime, not in the older one. Use 6.8

Reply:
and likely you don't need to build ffmpeg anyway.

Reply:
OK, thanks! I'm trying a build with 6.8 and ffmpeg-full.

Reply:
Unfortunately, same problem as before with 6.8 and ffmpeg-full: no h265 decoding and no h264 hardware decoding.

Reply:
Haruna doesn't use the ffmpeg-full (or ffmpeg) extension either: https://github.com/flathub/org.kde.haruna/blob/master/org.kde.haruna.yml

Reply:
two wrongs don't make a right.

Reply:
Well, I couldn't find a libmpv-based player which uses successfully that extension. I've used the manifests of Haruna and [Memento](https://github.com/flathub/io.github.ripose_jp.Memento/blob/master/io.github.ripose_jp.Memento.yml) to make this one.

Reply:
I've tried with `org.kde.Platform` `6.8` with the extension `org.freedesktop.Platform.ffmpeg-full` `24.08`.

Reply:
> Well, I couldn't find a libmpv-based player which uses successfully that extension. I've used the manifests of Haruna and [Memento](https://github.com/flathub/io.github.ripose_jp.Memento/blob/master/io.github.ripose_jp.Memento.yml) to make this one.

https://github.com/flathub/io.github.hydrusnetwork.hydrus/blob/949af31813e1700c7daf2fd9591ef112384a333d/io.github.hydrusnetwork.hydrus.yml#L10-L14



https://github.com/flathub/com.github.rafostar.Clapper/blob/f58477c35674f49aedc343c25d227f40eecf92f7/com.github.rafostar.Clapper.json#L7-L8